doc/developer/getting-started/index.md

```diff
@@ -109,6 +109,28 @@ bash install-libs.sh
 * Create a test database in MariaDB with the name `irida_test` and user `test` with password `test`.
 * Create a second test database in MariaDB with the name `irida_integration_test` and user `test` with password `test` (for running local integration tests).
 
+```
+# Setup databases example
+# These steps may differ depending on your installation
+
+# log into mysql as root
+sudo mysql -u root
```

Comment:
Can you add something separating the bash parts from the mysql parts?  Maybe break it into  separate code blocks.  Also that way you'll get proper syntax highlighting for the bash & mysql parts. 

```bash
# log into mysql as root
sudo mysql -u root
```
```mysql
# create databases
create database irida_test;
create database irida_integration_test;
...
```

If you didn't know, you can give hits to markdown about your language like so:
\`\`\`mysql
select code from comment;
\`\`\`

Reply:
Any updates on this?

Reply:
Should be fixed now. (I'm not sure how your comments here flew under my radar, sorry)